client/src/components/auth/AuthActions.js

```diff
@@ -26,14 +26,13 @@ const checkinUserAction = ({ email, password }) => dispatch =>
   axios
     .post('/users/login', { email, password })
     .then((response) => {
-      const message = response.data.message;
       const token = response.data.token;
       localStorage.setItem('token', token);
       const payload = API_URL === '9999'
```

Comment:
I use it to separate  my code for testing purposes. 
I declared my API_URL globally therefore each time my test run, it picks the URL for development and also tries decoding the token which for my tests won't work. I had to declare a separate API_URL in my jest config.js.
This line of code checks if the API_URL set is for the test environment or not. If it is, it sends the email and password I provided for testing else it proceeds with decoding the token gotten  to be further used in my application.